src/book/03-guides/04-general/02-git.mdx

```diff
@@ -36,7 +36,7 @@ Once you have a set of commits on your branch which you think should be added to
 
 GitHub has [documentation about PRs here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
 
-Title your PR with a description of the changes that it will make. For this example we could use `Add stand-up script`. If you need to give more detail about what the code does, put that as a comment.
+The title of your PR is important. It will determine what the eventual commit message will be that others will read to know what has changed. Title your PR with a description of the changes that it will make. For this example we could use `Add stand-up script`. If you need to give more detail about what the code does, put that as a comment. If done properly, your PR title should finish the sentence "**When merged, this PR will `<TITLE HERE>`**".
```

Comment:
```suggestion
The title of your PR is important. It will determine what the eventual commit message will be that others will read to know what has changed. Title your PR with a description of the changes that it will make. For this example we could use `Add a stand-up script`. If you need to give more detail about what the code does, put that as a comment. If done properly, your PR title should finish the sentence "**When merged, this PR will `<TITLE HERE>`**".
```